bitnami/mongodb/templates/arbiter/statefulset.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
   annotations: {{- include "common.tplvalues.render" (dict "value" .Values.arbiter.annotations "context" $) | nindent 4 }}
   {{- end }}
 spec:
-  serviceName: {{ include "mongodb.fullname" . }}-arbiter-headless
+  serviceName: {{ include "mongodb.arbiterservicename" . }}
```

Comment:
Change arbiterservicename to arbiter.service.nameOverride

Reply:
Fixed